.github/workflows/build.yaml

```diff
@@ -0,0 +1,54 @@
+name: Build
+
+on:
+  workflow_call:
+
+jobs:
+  build-charm-under-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
I strongly recommend you pin all your actions to specific versions to avoid unnoticed changes in the builds that, if problematic, are very hard to pinpoint

Reply:
Fixed in #499